.github/workflows/build-assets.yml

```diff
@@ -330,8 +330,7 @@ jobs:
           build-args: |
             PKG_FILE=${{ steps.ln-container.outputs.k8s_pkg_file }}
           tags: |
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.internal_tag }}
-            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/ln/platform-core-${{ matrix.os }}:${{ needs.preamble.outputs.candidate_base_branch }}
+            ${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/${{vars.REPOSITORY}}/hpcc-platform/${{ matrix.os }}/platform-core-ln:${{ needs.preamble.outputs.internal_tag }}
```

Comment:
Probably should just use 'secrets.REPOSITORY' for this.  And add it so there's a default built in like with dummy.io.

```
${{ secrets.JFROG_REGISTRY || 'dummy.io' }}/${{ secrets.REPOSITORY || 'hpccpl-docker-local' }}/hpcc-platform/${{ matrix.os }}/platform-core-ln:${{ needs.preamble.outputs.internal_tag }}
```

Reply:
sounds good

Reply:
However, I didn't want to hard code the repo name to avoid someone who actually have access to it to accidentally push custom builds there.

Reply:
You don't have to worry about that, hard-coding it to hpccpl-docker-local is fine.  This line above it will stop the push from occurring if anyone runs this code and is not the 'hpcc-systems' repository.

https://github.com/hpcc-systems/HPCC-Platform/pull/17908/files#diff-1cb3dcabfd7d99ae883acdccc1e1729299d1245fb8607a102fccf3868aa05158R329